src/lib.rs

```diff
@@ -233,14 +237,6 @@ pub fn check_verify_callback<T: ZcashScript, U: ZcashScript>(
     )
 }
 
-/// Convert errors that don’t exist in the C++ code into the cases that do.
-pub fn normalize_error(err: Error) -> Error {
```

Comment:
If this was in the public API for a previous release, document this removal in the changelog.